src/app/components/search/search-results/search-results.component.ts

```diff
@@ -114,21 +114,62 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
     }
   }
 
+  private isOperator(input: string): boolean {
+    if (input) {
+      input = input.trim().toUpperCase();
+
+      const operators = ['AND', 'OR'];
+      return operators.includes(input);
+    }
+    return false;
+  }
+
+  private formatFields(fields: string[], term: string): string {
+    let prefix = '';
+    let suffix = '*';
+
+    if (term.startsWith('=')) {
+      prefix = '=';
```

Comment:
what about prohibit operator `-` ,  required operator `+` ?

Reply:
I think that will go into a separate task/ticket. There's much more stuff to support in the long run https://docs.alfresco.com/6.0/concepts/rm-searchsyntax-intro.html